app/segments/drill/[id]/submission/input.js

```diff
@@ -376,8 +434,17 @@ function createOutputData(drillInfo, inputValues, attemptShots, uid, did) {
                 1;
               break;
             case "putt":
-              shot.strokesGained =
-                attemptShots[j].baseline - inputValues[j].strokes;
+              switch (drillInfo.requirements[0].type) {
+                case "inputtedPutt":
```

Comment:
Can you move all of this inputtedPutt specific code outside of the loop and handle all of them there so the code in the loop stays the same? Namely attemptShot[j].baseline and attemptShot[j].item.target can be at the start of the j loop